source/main.cpp

```diff
@@ -243,6 +242,12 @@ void GameLoop(PlayerInfo &player, const Conversation &conversation, const string
 	// IsDone becomes true when the game is quit.
 	while(!menuPanels.IsDone())
 	{
+		if(!conversation.IsEmpty() && dataFinishedLoading && !conversationLoaded)
```

Comment:
The order of these is intentional.  `!conversation.IsEmpty()` will always return `false` if the `--talk` argument is not used.